### DISCUSSION TITLE
Cursor's coordinates not fully displayed in Ubuntu

### DISCUSSION THREAD

**Discussion by thkfighter** [openTCS/opentcs#65] [General]
openTCS-5.6.0-bin

Model Editor in Ubuntu 20.04
The coordinates of the cursor are not fully displayed on the right of the bottom bar of the window.
![image](https://user-images.githubusercontent.com/28823037/204985247-263374a7-1f03-40b6-9fac-dcad1c346d3b.png)

Model Editor in Windows 11
The coordinates of the cursor are displayed in the right way.
![image](https://user-images.githubusercontent.com/28823037/204985329-e08c9cd6-109c-4d17-af94-d30b40542433.png)

**Comment by swltr** [comment]
@thkfighter For bug reports, please open issues in the future, not discussions.

I just tried this on Ubuntu 20.04, and it was displayed correctly with me. Which desktop environment did you use? (In my case, it was MATE.)

**Comment by sebo001** [comment]
Hi @thkfighter, does this issue still persist for you?
If so, it would be great if you could comment on what desktop environment you are using in Ubuntu 20.04. so that the issue can be reproduced.

**Comment by swltr** [comment]
@thkfighter FYI: The GUI layout has been improved with bfc0b9443a3cf3530c7a15ecd0f189a90be99547, which will be part of the next release.
